src/chat/components/ViewUnRead.tsx

```diff
@@ -14,6 +14,7 @@ interface ViewUnReadProps {
   customTextStyle?: StyleProp<TextStyle>;
   unReadSentMessage?: string;
   unReadSeenMessage?: string;
+  viewUnReadCustom?: (hasUnread: boolean) => JSX.Element;
```

Comment:
I think the name of this prop should be `customUnReadView` or `customViewUnRead`